lib/admin/parsers/workbook.rb

```diff
@@ -13,31 +17,49 @@ def initialize(rubyxl_workbook,
       end
 
       def call
-        as_hashes = transform_worksheet
+        [ parse_mediators, processed_blacklist ]
+      end
+
+      private
+
+      def parse_mediators
+        as_hashes = transform_mediators
         as_hashes = @data_parser.parse(as_hashes) if @data_parser
         as_hashes
       end
 
-      private
+      def parse_blacklist
+        blacklist_worksheet[1..-1].map { |row| row.cells.first.value }
```

Comment:
The heading 'Confidential' isn't used to establish which column the blacklist values are in so it's currently redundant.